source/hash_table.c

```diff
@@ -758,10 +758,7 @@ static inline void s_get_next_element(struct aws_hash_iter *iter, size_t start_s
 struct aws_hash_iter aws_hash_iter_begin(const struct aws_hash_table *map) {
     AWS_PRECONDITION(aws_hash_table_is_valid(map));
     struct hash_table_state *state = map->p_impl;
-    struct aws_hash_iter iter;
-    AWS_ZERO_STRUCT(iter);
-    iter.map = map;
-    iter.limit = state->size;
+    struct aws_hash_iter iter = {.map = map, .limit = state->size};
```

Comment:
Do we really need to make this change even? CBMC nowadays has field-sensitive constant propagation.

Reply:
Done.